index.js

```diff
@@ -52,14 +52,15 @@ const handleArguments = (file, args, options = {}) => {
 
 	options.env = getEnv(options);
 
-	options.stdio = normalizeStdio(options);
+	const stdio = normalizeStdio(options);
+	options.stdio = transformStdio(stdio);
```

Comment:
We cannot directly convert iterables to a stream because `child_process.spawn()` only allows streams that have a file descriptor.

Instead, we pass `stdin: 'pipe'`. Once the process is spawned, we create a stream from the iterable and pipe it onto `spawned.sdin`.

`options.stdio` is the actual value passed to `child_process.spawn()` (i.e. `pipe`) while `parsed.stdio` keeps track of the original iterable passed as argument.